### DISCUSSION TITLE
Conference Room structure

### DISCUSSION THREAD

**Discussion by braysonjohn148** [ant-media/Ant-Media-Server#6271] [Q&A]
I am using Flutter, but i am creating a room using Rest API. => **/broadcasts/create**

I want when a host adds another person to be the speaker in the room. The viewers do not get any update that there is a new streamer in the room.

Or i should create a conference room, and how do i get to add broadcasters to the conference

**Comment by zerostack0** [comment]


**Reply by braysonjohn148** [reply]
Got it @mustafaboleken , Thank you for the reply. Waiting
